.gitignore

```diff
@@ -38,3 +38,5 @@ tailscale.jks
 libtailscale.aar
 libtailscale-sources.jar
 .DS_Store
+
+tailscale.version
```

Comment:
I thought we were going to check this file in to the repo? Or did you not `git add` it?